src/components/common/UserCard/ui-card-content/CardContent.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from './CardContent.module.css';
+
+export default function CardContent({ questionCount }) {
+  return (
+    <div className={styles.wrap}>
+      <div className={styles.questionWrap}>
+        <img className={styles.messagesIcon} src='/src/images/icons/messages.svg' alt='' />
+        <span>받은 질문</span>
+      </div>
+      <span>{(questionCount = '9개')}</span>
```

Comment:
```suggestion
      <span>{questionCount}</span>
```
값이 화면에 잘 나오는지 확인해 보기위한 부분인 것 같습니다!
테스트가 끝나면 다시 원 상태로 복구 시켜 주세요~

Reply:
수정했습니다.